SDPi_Supplement/asciidoc/docinfo.html

```diff
@@ -7,7 +7,7 @@
 }
 
 #toc.toc2 {
-    width: 30%;
+    width: 30em;

```

Comment:
Is there a reason to change this to a fixed value? This breaks the responsive design I enabled for the document.

Reply:
When it was a %, it was clipping off the document's left columns.  So reverting back to a fixed width fixed the one problem but in a sub-optimal way.   I believe the plan was to do it The Right Way somewhere down the line.   For example, as I mentioned in Session #1 yesterday, eventually we would like to include all levels in an outline that collapses to the top level or two.